infra/testing/sw-env.js

```diff
@@ -46,3 +49,4 @@ global.location = 'https://example.com';
 global.registration.sync = new SyncManager();
 global.Request = Request;
 global.SyncEvent = SyncEvent;
+global.URLSearchParams = URLSearchParams;
```

Comment:
May need to double check Edge support for this. It's not noted on MDN.

Reply:
I can try to ask some of the Edge folks if they're planning to implement this with service worker. I wouldn't want to add code to do this just out of fear that it wouldn't work once Edge ships.

Reply:
Looks like Jake already made this point: https://wpdev.uservoice.com/forums/257854-microsoft-edge-developer/suggestions/16856668-support-the-searchparams-property-of-url-objects

Reply:
Based on https://twitter.com/patrickkettner/status/921144430299512832, let's keep this as-is and re-evaluate how to patch/polyfill for Edge in the event it ships SW without URLSearchParams.

Reply:
And this: https://twitter.com/dstorey/status/921211253623701505